ui/js/pods-dfv/_src/text/text.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class PodsDFVText extends React.Component {
+	constructor ( props ) {
```

Comment:
Could this type of stuff be put in a new PodsDFVReactField type of class where it does the most common stuff and you just extend it to customize?